comet/broker.py

```diff
@@ -36,10 +37,11 @@
 from .manager import Manager, TIMESTAMP_FORMAT
 from .exception import CometError, DatasetNotFoundError, StateNotFoundError
 
-
+REDIS_PORT = os.environ['REDIS_PORT']
+REDIS_HOST = os.environ['REDIS_HOST']
```

Comment:
In normal operation, these environmental variables may not be in the host environment. To ensure a default value, you can replace these calls with `os.environ.get("REDIS_PORT", DEFAULT_PORT)` (and equivalent for host).

Reply:
Thanks Adam! Will do that.